Integrations/AttackIQFireDrill/AttackIQFireDrill.py

```diff
@@ -345,16 +351,28 @@ def get_test_results_command(args=demisto.args()):
     """ Implements attackiq-get-test-results command
     """
     test_id = args.get('test_id')
+    page, page_size = get_page_number_and_page_size(demisto.args())
     params = {
-        'page': args.get('page', '1'),
+        'page': page,
+        'page_size': page_size,
         'test_id': test_id,
         'show_last_result': args.get('show_last_result') == 'True'
     }
     try:
         raw_test_res = http_request('GET', '/v1/results', params=params)
         test_res = build_transformed_dict(raw_test_res['results'], TEST_RESULT_TRANS)
+        test_cnt = raw_test_res.get('count')
+        total_pages = math.ceil(test_cnt / page_size)
```

Comment:
<img width="160" alt="image" src="https://user-images.githubusercontent.com/19407287/64921560-60929f00-d7cd-11e9-8191-b8800766359f.png">

I think we have a problem with this devision

Reply:
irrelevant